xdem/spstats.py

```diff
@@ -68,7 +68,7 @@ def interp_nd_binning(df: pd.DataFrame, list_var_names: Union[str,list[str]], st
 
     # Extrapolated linearly outside the 2D frame.
     >>> fun((-1, 1))
-    array(-5.)
+    array(-1.)
```

Comment:
weird, it worked for me with the other value!

Reply:
Huh!

Reply:
oh yes I fixed a bug which affected the dimension order since then, when writing the tests.
so this is now the result along the `x` dimension, but before it was along `y`: if you try `fun((1,-1))` you'll get `array(-5.)`.

So all good! And I just exemplified why we need to add the doctests! :sweat_smile:

Reply:
Great!